ouroboros-consensus-byron/test/Test/Consensus/Byron/Ledger/Golden.hs

```diff
@@ -463,15 +478,396 @@ test_golden_LedgerState = goldenTestCBOR
+    , TkInt 2000000
+    , TkInt 8192
+    , TkInt 700
+    , TkInt 10000000000000
+    , TkInt 5000000000000
+    , TkInt 1000000000000
+    , TkInt 100000000000000
+    , TkInt 10
+    , TkListLen 3
+    , TkInt 900000000000000
+    , TkInt 600000000000000
+    , TkInt 50000000000000
+    , TkListLen 2
+    , TkInt 0
+    , TkTag 24
+    , TkBytes "\130\ESC\NUL\NUL\141QuOR\NUL\ESC\NUL\NUL\NUL\n>\154\184\NUL"
+    , TkInteger 18446744073709551615
+    , TkListBegin
+    , TkBreak
+    , TkMapLen 0
+    , TkMapLen 0
+    , TkMapLen 0
+    , TkMapLen 0
+    , TkMapLen 0
+    , TkTag 258
+    , TkListLen 0
+    , TkMapLen 0
+    , TkListLen 2
+    , TkListLen 2
+    , TkListBegin
+    , TkBreak
+    , TkTag 258
+    , TkListLen 4
+    , TkListLen 2
+    , TkInt 0
+    , TkBytes "\SUB\US\247\ETXQ\ETX\216\169\225\229jNN\GS\NUL1\251?1\166\169\149\&17\223\229Lg"
+    , TkListLen 2
+    , TkInt 0
+    , TkBytes "(\RSZ\233\227W\151\n\r\208\a\220\155!gZ\176}\227\214~\200%\215\&66!\249"
+    , TkListLen 2
+    , TkInt 0
+    , TkBytes "D(<\229\196Nn\NUL\132a\144\173_\180p\197\219\220gL\"\223*>\145\\Mq"
+    , TkListLen 2
+    , TkInt 0
+    , TkBytes "_S\224\RS\DC3f\174\218\136\DC1\194\166\&0\240\224\&7\az{e\DLE\147\210\189\196\239r\NUL"
+    , TkListLen 2
+    , TkListBegin
+    , TkListLen 2
+    , TkBytes "\SUB\US\247\ETXQ\ETX\216\169\225\229jNN\GS\NUL1\251?1\166\169\149\&17\223\229Lg"
+    , TkBytes "\179\f>\t\FS\209\143v*\151RJlV$\195\251\FS\224\t;P*\n0\DC2\248\242"
+    , TkListLen 2
+    , TkBytes "(\RSZ\233\227W\151\n\r\208\a\220\155!gZ\176}\227\214~\200%\215\&66!\249"
+    , TkBytes "\223I\SOH\232\246\171\217j\139\208W\159$\ETXj\139\155$\193\223K\237\133\161@-\176\155"
+    , TkListLen 2
+    , TkBytes "D(<\229\196Nn\NUL\132a\144\173_\180p\197\219\220gL\"\223*>\145\\Mq"
+    , TkBytes "\r\145eg\249kje\210\EOT\150nj\171_\189$.V\195!\131?\139\165\214\a\218"
+    , TkListLen 2
+    , TkBytes "_S\224\RS\DC3f\174\218\136\DC1\194\166\&0\240\224\&7\az{e\DLE\147\210\189\196\239r\NUL"
+    , TkBytes "\248>\158\STXp\171\204tK&W\v\ACK\159f\219Z\131\205\235\228\190\219|\212\&5\196C"
+    , TkBreak
+    , TkMapLen 4
+    , TkBytes "\SUB\US\247\ETXQ\ETX\216\169\225\229jNN\GS\NUL1\251?1\166\169\149\&17\223\229Lg"
+    , TkInt 0
+    , TkBytes "(\RSZ\233\227W\151\n\r\208\a\220\155!gZ\176}\227\214~\200%\215\&66!\249"
+    , TkInt 0
+    , TkBytes "D(<\229\196Nn\NUL\132a\144\173_\180p\197\219\220gL\"\223*>\145\\Mq"
+    , TkInt 0
+    , TkBytes "_S\224\RS\DC3f\174\218\136\DC1\194\166\&0\240\224\&7\az{e\DLE\147\210\189\196\239r\NUL"
+    , TkInt 0
+    , TkListLen 2
+    , TkListLen 1
+    , TkInt 0
+    , TkListLen 0
+    , TkListLen 3
+    , TkListLen 2
+    , TkInt 0
+    , TkListLen 3
+    , TkListLen 0
+    , TkListLen 0
+    , TkListLen 1
+    , TkInt 0
+    , TkListLen 0
+    , TkListLen 0
+    ]
+
+  where
+    exampleExtLedgerState = ExtLedgerState
+      { ledgerState = exampleLedgerState
+      , headerState = genesisHeaderState S.empty
```

Comment:
I'm thinking now that we should best store at least one `AnnTip` in here, as otherwise we wouldn't notice changes to that type, even though they'd change the format.